src/transformers/models/bert/modeling_bert.py

```diff
@@ -478,10 +473,9 @@ def forward(
             cross_attn_past_key_value = past_key_value[-2:] if past_key_value is not None else None
             cross_attention_outputs = self.crossattention(
                 attention_output,
-                attention_mask,
+                encoder_attention_mask,
```

Comment:
This looks good to me. We should also remove the `encoder_attention_mask` argument from `BertSelfAttention`, `RobertaAttention` etc because we are passing positional arguments here, and `output_attentions` is being assigned to `past_key_values`, which is breaking the tests.